docs/TROUBLESHOOT.md

```diff
@@ -0,0 +1,32 @@
+# TROUBLESHOOT
+
+
+## Common Stellio API mistakes 
+
+### Sending a single entity to a entityOperations endpoint
+
```

Comment:
Add an intro sentence like : "when calling an entityOperations endpoints, if you send ..."